### PR TITLE
Modification performed in 'blog.css' file

### PR DESCRIPTION
**Pull request by Kavs-123**

### CODE REVIEW

File: css/blog.css [hunk 1]
```diff
@@ -15,71 +16,22 @@
   font-family: Montserrat, sans-serif;
   font-weight: 500;
   padding: 10px 10px;
-  transition: all 300ms ease-in-out;
```
Comment (ArunTeltia) [inline css/blog.css:18]: Add this back

File: css/blog.css [hunk 2]
```diff
@@ -342,9 +247,9 @@ body.no-max-width .site-content {
 
 @media only screen and (min-width: 61.063em) {
   body.layout-one-column-narrow .site-content {
```
Comment (ArunTeltia) [inline css/blog.css:249]: Remove this as well

File: css/blog.css [hunk 3]
```diff
@@ -354,44 +259,37 @@ body.no-max-width .site-content {
...
 
-.content-area {
-  width: 63.88889%;
-  float: left;
-  margin-left: 1.38889%;
-  margin-right: 1.38889%;
-}
-
 @media only screen and (max-width: 61.063em) {
   .content-area {
```
Comment (ArunTeltia) [inline css/blog.css:263]: remove this as well

File: css/blog.css [hunk 4]
```diff
@@ -354,44 +259,37 @@ body.no-max-width .site-content {
...
-    margin-left: 1.38889%;
-    margin-right: 1.38889%;
+     width: 97.22222%;
+     float: left;
+     margin-left: 1.38889%;
+     margin-right: 1.38889%;
   }
 }
 
 #tertiary {
```
Comment (ArunTeltia) [inline css/blog.css:287]: remove this as well

**Review by ArunTeltia — CHANGES_REQUESTED** [review]

**Review by yasharth291 — APPROVED** [review]

**Comment by ArunTeltia** [comment]
I tested @LordShenron Can be merged, Size down by (0.05mb)

**Pull request merged**